build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
I'd suggest keeping Kokoro-related files under Kokoro folder, e.g. `kokoro/scripts`

Reply:
This file still exists.